src/runner.ts

```diff
@@ -46,6 +46,8 @@ export class Runner {
 
   constructor(config: Config, reporter: Reporter) {
     this._config = config;
+    // Resolve symlinks. TODO: do this asynchronously?
+    this._config.testDir = fs.realpathSync(this._config.testDir);
```

Comment:
I think this is unexpected, user wants to see their logical path, not where it resolves to.